src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -61,7 +61,7 @@ object Boot extends App {
                     "rawls-service")
 
     system.actorOf(SubmissionSupervisor.props(
-      new GraphSubmissionDAO(),
+      new GraphSubmissionDAO(new GraphWorkflowDAO()),
```

Comment:
no change needed, but is there a reason we would need to pass a DAO to another DAO in a constructor - can't a DAO just instantiate one as needed - they aren't tied to anything, the functions themselves use transactions